### PR TITLE
feat: add project's link in the markdown format for list of tools

### LINKED ISSUE #845: Add links to projects in the Catalog of CLIs section of the README
## Expected Behaviour

I'd like to be able to link to each project using the URL we have for the tool, when generating the markdown output for the README

## Current Behaviour

Not available right now.

## Possible Solution

Output a URL from the tool's definition?

### PR DESCRIPTION
**Pull request by Jasstkn**
Signed-off-by: Maria Kotlyarevskaya <mariia.kotliarevskaia@gmail.com>

## Description

- for markdown output adds formatting to make name of the tool clickable link
- for table output adds a separate columnt with project's URL
- use switch/case to make code clear as it requires use of format in few places

## Motivation and Context

- [x] I have raised an issue to propose this change, which has been given a label of `design/approved` by a maintainer ([required](https://github.com/alexellis/arkade/blob/master/CONTRIBUTING.md))
fixes #845 

## How Has This Been Tested?

The result of `arkade get --output markdown` is added in this PR in the `README.md`

```
./arkade get --output table
...
+------------------+-------------------------------------------------------------+--------------------------------------------------------------+
| viddy            | https://github.com/sachaos/viddy                            | A modern watch command. Time machine and pager etc.          |
+------------------+-------------------------------------------------------------+--------------------------------------------------------------+
| waypoint         | https://github.com/hashicorp/waypoint                       | Easy application deployment for Kubernetes and Amazon ECS    |
+------------------+-------------------------------------------------------------+--------------------------------------------------------------+
| yq               | https://github.com/mikefarah/yq                             | Portable command-line YAML processor.                        |
+------------------+-------------------------------------------------------------+--------------------------------------------------------------+
There are 119 tools, use `arkade get NAME` to download one.

```

## Are you a GitHub Sponsor yet (Yes/No?)

- [ ] Yes
- [x] No

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Documentation

- [x] I have updated the list of tools in README.md if (required) with `./arkade get -o markdown`
- [ ] I have updated the list of apps in README.md if (required) with `./arkade install --help`

## Checklist:
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/alexellis/arkade/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`

- [ ] I have tested this on arm, or have added code to prevent deployment

### CODE REVIEW

**Comment by alexellis** [comment]
I was thinking for the markdown format, that the links may be actual anchor tags `<a></a>` so `[Project name](https://github.com/....)

This would keep the width of the markdown table down and also mean it will work on markdown renderers that don't convert all URLs to anchor tags. Thoughts?

**Comment by Jasstkn** [comment]
> I was thinking for the markdown format, that the links may be actual anchor tags `<a></a>` so `[Project name](https://github.com/....)
> 
> This would keep the width of the markdown table down and also mean it will work on markdown renderers that don't convert all URLs to anchor tags. Thoughts?

Sounds good. Let me try it out!

File: pkg/get/table.go [hunk 1]
```diff
@@ -18,23 +18,35 @@ const (
...
+		table.SetColumnColor(tablewriter.Colors{tablewriter.Bold, tablewriter.FgGreenColor}, tablewriter.Colors{}, tablewriter.Colors{})
 	}
 
 	for _, t := range tools {
-		table.Append([]string{t.Name, t.Description})
+		url := fmt.Sprintf("https://github.com/%s/%s", t.Owner, t.Repo)
+
+		switch format {
+		case MarkdownStyle:
+			name := fmt.Sprintf("[%s](%s)", t.Name, url)
```
Comment (alexellis) [inline pkg/get/table.go:45]: Wait, it looks like you may have done this already.

I think I was confused by the example in the testing section being for the plain format table.

Do we know that every link in arkade has an equivalent URL on GitHub?

**Comment by alexellis** [comment]
I just commented again, I think you've already done this

**Review by alexellis — APPROVED** [review]
Approved

File: pkg/get/table.go (hunk 1 above)
Comment (Jasstkn) [reply pkg/get/table.go:45]: Yes, I think I've clicked all of them...

Comment (Jasstkn) [reply pkg/get/table.go:45]: I think I misunderstood you. But yes, i've done the markdown links for markdown format, separate column for table view and no additional changes just to the list of tools.

**Pull request merged**